strings/tcg/set_one.json

```diff
@@ -61,74 +61,74 @@
 			"id": "assistant",
 			"name": "Staff Assistant",
 			"desc": "The lowest ladder on the Nanotrasen Employment Ladder, Staff Assistants are employed to help out with tasks deemed 'too menial for robots'.",
-			"rules": "{$Graytide}, for every card with '{$Graytide}', this card has +1/+1 on the field.",
+			"rules": "{$Graytide}. The buff this card gains from Graytide is doubled.",
 			"icon_state": "assistant",
 			"power": "1",
 			"resolve": "1",
 			"faction": "Service",
 			"summoncost": "1",
 			"cardtype": "Creature",
-			"cardsubtype": "Human Employee",
+			"cardsubtype": "Maintenance Service",
 			"rarity": "common",
 			"summon_icon_state": "Assistant"
 		},
 		{
 			"id": "atmos_tech",
 			"name": "Atmospheric Technician",
 			"desc": "The Atmospheric Technicians are tasked with keeping the station's air clean, breathable, and, most importantly, devoid of plasma.",
-			"rules": "{$On Summon}: Search your deck for an Atmospherics Battlefield card, and add it to your hand. Shuffle your deck afterward.",
+			"rules": "{$On Summon}: Search your deck for a Battlefield card, and add it to your hand. Shuffle your deck afterward.",
 			"icon_state": "atmos_tech",
-			"power": "2",
-			"resolve": "3",
+			"power": "3",
+			"resolve": "2",
 			"faction": "Engineering",
-			"summoncost": "4",
+			"summoncost": "2",
 			"cardtype": "Creature",
-			"cardsubtype": "Human Engineer",
+			"cardsubtype": "Atmosian Engineering",
 			"rarity": "common",
 			"summon_icon_state": "Atmospheric Technician"
 		},
 		{
 			"id": "bartender",
 			"name": "Bartender",
 			"desc": "Prior to the introduction of on-station psychologists, the Bartender served to alleviate many employees' woes and fears. Remember, always drink responsibly.",
-			"rules": "",
+			"rules": "All your other creatures gain +1 power.",
 			"icon_state": "bartender",
-			"power": "3",
+			"power": "2",
 			"resolve": "2",
 			"faction": "Service",
-			"summoncost": "3",
+			"summoncost": "2",
 			"cardtype": "Creature",
-			"cardsubtype": "Human Employee",
+			"cardsubtype": "Service Bar",
 			"rarity": "common",
 			"summon_icon_state": "Bartender"
 		},
 		{
 			"id": "botanist",
 			"name": "Botanist",
 			"desc": "The Botanist is in charge of keeping the station's food supply happy, healthy, and preferably not laced with hallucinogens.",
-			"rules": "",
+			"rules": "Start of turn: Heal 1 healthshard.",
 			"icon_state": "botanist",
-			"power": "1",
-			"resolve": "4",
+			"power": "0",
+			"resolve": "3",
 			"faction": "Service",
 			"summoncost": "1",
 			"cardtype": "Creature",
-			"cardsubtype": "Human Employee",
+			"cardsubtype": "Service Bar Plants",
```

Comment:
Potential consistency issue: using both a keyword in **Plants** type card for botanist cards and **Plant** cards for literal plants may cause issues carrying forward, for example a plant card being able to tutor out a plants card but not vis versa.

Reply:
Was a mistake - meant to make the botanist just a Plant card, not Plants. Fixing that